### DISCUSSION TITLE
Enum constants are rendered with white text

### DISCUSSION THREAD

**Discussion by navaronbracke** [community/community#56307] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When viewing enum constants, which have doc comments,
the value of the constant is rendered using a white font color.
It works properly when the enum has no doc comment.

Reproduced with Dart code & DartDoc comments (`///`), not sure if this is a more general problem.

Repro:
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/media_query.dart#L36
https://github.com/spencerccf/app_settings/blob/master/lib/src/app_settings_panel_type.dart#L4

**Comment by AdamShwert** [comment]
We have a fix for this which should hopefully be deploying soon. Thank you for the report!

**Reply by AdamShwert** [reply, marked as answer]
The fix should now be deployed.
